src/distributors/adoptopenjdk-installer.ts

```diff
@@ -0,0 +1,79 @@
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+
+import fs from 'fs';
+import path from 'path';
+import semver from 'semver';
+
+import { IS_WINDOWS, PLATFORM, IS_MACOS, macOSJavaContentDir } from "../util";
+import { JavaInstallerResults, IJavaRelease, JavaBase, JavaInstallerOptions } from "./base-installer";
+import { IRelease, IReleaseVersion } from './adoptopenjdk-models'
+
+export class AdoptOpenJdkDistributor extends JavaBase {
+    
+    constructor(initOptions: JavaInstallerOptions) {
+        super("AdoptOpenJDK", initOptions);
+    }
+
+    private async getAvailableMajor(range: semver.Range) {
```

Comment:
let's name this function as `resolveMajorVersion` and move to bottom

Reply:
Done.